modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderLocalServiceImpl.java

```diff
@@ -332,11 +335,6 @@ public CommerceOrder addCommerceOrder(
 			_setAccountDefaultBillingAddress(commerceChannel, commerceOrder);
 		}
 
-		commerceOrder.setCommerceAccountId(commerceAccountId);
-		commerceOrder.setCommerceCurrencyCode(commerceCurrencyCode);
-		commerceOrder.setCommerceOrderTypeId(commerceOrderTypeId);
-		commerceOrder.setCommerceShippingMethodId(commerceShippingMethodId);
-
```

Comment:
`_setAccountDefaultBillingAddress` and `_setAccountDefaultShippingAddress` both need to be after 
`commerceOrder.setCommerceAccountId(commerceAccountId);`

For clean look purposes, I moved both checks down.